packages/react-core/src/components/Wizard/WizardToggle.tsx

```diff
@@ -91,7 +97,11 @@ export const WizardToggle: React.FunctionComponent<WizardToggleProps> = ({
             aria-label={mainAriaLabel}
             aria-labelledby={mainAriaLabelledBy}
             hasNoBodyPadding={hasNoBodyPadding}
+            activeStep={activeStep}
+            isDrawerExpanded={isDrawerExpanded}
+            hasDrawer={hasDrawer}
           >
+            {!isDrawerExpanded && activeStep.drawerToggleButton}
```

Comment:
We should also check for `hasDrawer` here so the button will not render if there is not drawer.